sphinx/application/network/concepts/queries/concept-network-query-save-profile/src/main/java/chat/sphinx/concept_network_query_save_profile/model/PersonInfoDto.kt

```diff
@@ -0,0 +1,15 @@
+package chat.sphinx.concept_network_query_save_profile.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PersonInfoDto(
+    val id: Int,
+    val host: String,
+    val owner_alias: String,
+    val description: String,
+    val img: String,
+    val tag: String?,
```

Comment:
Should be `tags` instead of tag and type should be Array of String

Reply:
updated in bb3d4b5